docs/Course Wiki/ECE Course Offerings/ECE374B.md

```diff
@@ -0,0 +1,66 @@
+# ECE374B
+## Introduction to Algorithms and Models of Computation
+
+ECE374B is one of two **independent** sections of CS/ECE374, Introduction to Algorithms and Models of Computation. It is unoficially the 'ECE' version of 374. It is considered by some to be easier than the A version.
```

Comment:
I like this we should add this to every page (like a little overview at the top)

Reply:
"unofficially" is spelled incorrectly.
No need for "Introduction to Algorithms and Models of Computation" header.
Probably remove the sentence about one is easier than the other. Instead, just say "Notes on the difference between the two sections can be found here" and link the taking X vs. Y page.